mrp_progress_button/views/production.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="mrp_production_progress_button_form_view" model="ir.ui.view">
```

Comment:
cosmetical change: use original id

> prefer naming the inherited view after the original over using a name which follows the guidelines

not relevant for approvement

Reply:
@tbaden Sorry did not get you. Would you please let me know the exact change I need to do?

Reply:
@sudhir-erpharbor the id should be 'mrp_production_form_view' instead of 'mrp_production_progress_button_form_view', but it's not required for the approve itself.
Will approve it, but would be a nice change to have. as it's the oca naming convention.